plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: PEXProtection
 author: "RevivalPMMP"
-api: [3.0.0-ALPHA12, 3.0.0-ALPHA13, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 3.0.5, 3.1, 4.0.0]
+api: [3.0.0, 4.0.0]
```

Comment:
Jason's comment was that the highest version supported should be 3.0.0. 

Just because this currently works with PocketMine-MP 4.0.0 versions, does not mean it will work with the 4.0.0 versions once they are released.

Please read this from DKTApps regarding release flow.
https://forums.pmmp.io/threads/release-flow-and-old-version-support.6132/